e2e/utils/pages/homepage.page.ts

```diff
@@ -22,7 +22,7 @@ interface HomepageParams {
 	viewportSize?: ViewportSize;
 }
 
-type HomepageLoggedInParams = {
+export type HomepageLoggedInParams = {
```

Comment:
Obviously not in this PR, but I guess it's time to extract shared between all pages things from `Homepage` into `Page` class and use it as a parent for all further pages.

Reply:
yes, makes sense! Included in my backlog